imports/ui/housingUnits/housingUnitsCreateView.js

```diff
@@ -12,7 +12,7 @@ Template.housingUnitCreateView.events(
       const inactive = tmpl.find('input[name="inactive"]:checked').value;
       const inService = tmpl.find('input[name="in_service"]:checked').value;
       const vacant = tmpl.find('input[name="vacant"]:checked').value;
-      const housingObject = {
+      const housingObject = [{
```

Comment:
can you explain why are you putting housingUnit object into an array?

Reply:
POST request accept an array, PUT request is not (see the logs). I will fix your issue and merge the branch.